genienlp/arguments.py

```diff
@@ -472,26 +480,29 @@ def parse_argv(parser):
         'ngrams: lookup all ngrams in the text and see if there is a match',
     )
 
-    parser.add_argument('--verbose', action='store_true', help='Print detected types for each token')
     parser.add_argument(
         '--almond_domains', nargs='+', default=[], help='Domains used for almond dataset; e.g. music, books, ...'
     )
     parser.add_argument(
         '--ned_features',
         nargs='+',
         type=str,
-        default=[],
-        help='Features that will be extracted for each entity. Order is important',
+        default=['type_id', 'type_prob', 'qid'],
+        help='Features that will be extracted for each entity: "type" and "qid" are supported. Order is important',
     )
     parser.add_argument(
         '--ned_features_size',
         nargs='+',
         type=int,
-        default=[],
+        default=[1, 1, 1],
         help='Max length of each feature vector. All features are padded up to this length',
```

Comment:
One part I still don't understand after all this time is the meaning of "feature vector". You have three features, and each feature is a sequence of words/classes coming out of Bootleg. What does "1" mean? What does it mean that the default value is 0 or 1.0?

Reply:
Also, what happens if the ned_ parameters given to `train` don't match those given to `run-bootleg`?

Reply:
Good catch. Since I moved out bootleg entirely out of almond task now, we don't need these arguments in `run_bootleg` anymore. I will remove them in this PR.

Reply:
The features are defined per token and include information about the wikidata qids, types, and probs assigned by bootleg . Each token can have up to 3 features defined by `ned_features`. Each of these features is a list of numbers. The size and default value for this list is defined by `ned_features_size` and `ned_features_default_val`.
For example, if size for type_id feature is 1, we only keep the first type associated with an entity Qid. 
If size for qid is 2, we keep the top 2 qids predicted by bootleg.
The default value is used to pad features up to the maximum size.